doc/release-notes.md

```diff
@@ -4,36 +4,83 @@ release-notes at release time)
 Notable changes
 ===============
 
+Changes to Testnet NU5 Consensus Rules
+--------------------------------------
+
+- In order to better support hardware wallets, transparent signature hash construction as
+  defined in [ZIP 244](https://zips.z.cash/zip-0244) has been modified to include a hash
+  of the serialization of the amounts of all outputs being spent, along with a hash of all
+  spent outputs `scriptPubKeys` values, except in the case that the `ANYONECANPAY` flag is
+  set. This allows hardware wallet devices to verify the UTXO amounts without having to
+  stream all the previous transactions containing the outputs being spent to the device.
+  Also as part of these changes, the transparent signature hash digest now commits
+  directly, rather than implicitly, to the sighash type, and the sighash type is
+  restricted to a fixed set of valid values. The change to ZIP 244 can be seen 
+  [here](https://github.com/zcash/zips/commit/ac9dd97f77869b9191dc9592faa3ebf45475c341).
+- This release fixes a bug in `v4.6.0` that caused a consensus failure on the Zcash
+  testnet at height `1,779,200`. For details see
+  <https://github.com/zcash/zcash/commit/5990853de3e9f04f28b7a4d59ef9e549eb4b4cca>.
+- There have been changes to the Halo2 proving system to improve consistency between the
+  specification and the implementation, and these may break compatibility. See for example
+  <https://github.com/zcash/halo2/commit/247cd620eeb434e7f86dc8579774b30384ae02b2>.
+- There have been numerous changes to the Orchard circuit impementation since `v4.6.0`.
+  See <https://github.com/zcash/orchard/issues?q=label%3AM-consensus-change-since-0.1.0-beta-1>
+  for a complete list.
+- A potential Faerie Gold vulnerability affecting the previous activation of NU5 on
+  testnet and existing since `v4.6.0` has been mitigated.
+
+NU5 Testnet Reactivation
+------------------------
+
+To support the aforementioned testnet consensus changes, the following changes are made in
+`zcashd v4.7.0`:
+
+- The consensus branch ID for NU5 is changed to `0xC2D6D0B4`.
+- The protocol version indicating NU5-aware testnet nodes is set to `170050`.
+- The testnet reactivation height for NU5 is set to **1,840,000**.
```

Comment:
Should this be a code escape, like `1,840,000`?

Reply:
This was copied from how it was written in the 4.5.0 PR.